style.css

```diff
@@ -0,0 +1,41 @@
+/* reset */
+* {
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+html {
+  height: 100%;
+  width: 100%
```

Comment:
If using the universal selector * to set width of all elements to 100% this line isn't doing anything.